packages/game/src/lib/find-duplicate-player-card.ts

```diff
@@ -5,9 +5,10 @@ import { PlayerLabel } from './player';
 export const findDuplicatePlayerCard = (
   board: ReadonlyDeep<Board>,
   playerLabel: PlayerLabel,
+  ownerLabel: PlayerLabel,
```

Comment:
Let's skip passing the owner altogether. We had everthing we needed before, but now we'll expand the `flatMap` to pull the string apart and compare `card` and `player` of the card.